mobile/src/common/enums/ui/z-indexes.enum.ts

```diff
@@ -0,0 +1,5 @@
+enum ZIndexes {
```

Comment:
enums' names must be singular

Reply:
btw now sure that we need this enum @Liza-Veis what do you think?

Reply:
Yes, I agree, it is used very rarely in react-native

Reply:
ok i will delete this enum